DP02_13a_Image_Cutout_SciDemo.ipynb

```diff
@@ -0,0 +1,1286 @@
+{
```

Comment:
<p>Is setting low S/N u-band objects' magnitudes to their errors effectively removing them from the sample? (i.e., they get really low umag values, so their u-g colors are like -25, so they don't get selected by the u-g color cut) If that's the case, why not just deselect all of those objects?</p><p> </p><p>I suspect you actually <em>want</em> the objects with low S/N? If that's the case, then you might want to set their mags artificially to 99.9 or something.</p><p> </p><p>I might just be misunderstanding, so maybe clarification would help.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/rubin-dp0/tutorial-notebooks/pull/171/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='42'/>

Reply:
<p>Well, its not really low u-band magnitude objects that I am altering, it is those with low  S/N in U (so like, if you measure a very red U-G color because you got 0 U-band flux, but the error was +/- 10, then thats not significant measure of such a red color). So, in that case I set the u-band to be the rms, like its an upper limit (and that usually makes those objects bluer in U-G because error &gt; flux). We only want to identify objects that we can significantly measure to be red in U-G. I will try to clarify this more, I agree it is a bit poorly explained</p>

Reply:
<p>Ok I tried to clarify this point now in terms of non-detections in u-band (i.e. upper limit in flux which is the rms of the photometry) translating into a lower limit in U-G color. Hopefully that is better?</p>